hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/CryptoService.java

```diff
@@ -57,4 +58,15 @@ static CryptoService getInstance() {
         return ServiceFactory.loadService(
                 CryptoService.class, ServiceLoader.load(CryptoService.class));
     }
+
+    /**
+     * Creates {@link AccountKeyLookup} object for other modules to uset during pre-txn handle
+     * phase. NOTE: This class can be modified to return other Account's fields as needed in the
+     * future.
+     *
+     * @param states states to use for looking up accounts
+     * @return lookup for account's keys
+     */
+    @NonNull
+    AccountKeyLookup createAccountKeyLookupFor(States states);
```

Comment:
I can imagine that this method becomes obsolete once we add `createStore()` as we have discussed yesterday.

Reply:
Agreed will delete this method.